packages/account/src/Types/common.type.ts

```diff
@@ -201,3 +201,13 @@ export type TIDVFormValues = {
     document_additional?: string;
     error_message?: string;
 };
+
+export type TServerError = {
+    code: string;
+    message: string;
+    details?: { [key: string]: string };
+    fields?: string[];
+};
+
+export const isServerError = (error: unknown): error is TServerError =>
```

Comment:
Lets not have utils functions in types file. Can u move them to `packages/account/src/Helpers/utils.ts`